qadence/backends/pulser/backend.py

```diff
@@ -250,7 +255,14 @@ def run_dm(
                 sequence = self.assign_parameters(circuit, param_values_el)
                 sim_result = simulate_sequence(sequence, self.config, state)
```

Comment:
Get the final statevector here, flip the order like in line 218 and then append it. This would match the description similar to the run method. I think then you need to change line 52 in PSRExepctation to not redo computing the final statevector there. 
Feel free to ignore this if you think its irrelevant. Just trying to get things to be uniform. Not sure where the exchange of 0,1 convention is included in this density matrix flow